cypress/integration/ozon-content.spec.ts

```diff
@@ -192,6 +192,21 @@ describe("Ozon Content", () => {
       .contains("document");
   });
 
+  it('should render ExtIoRef element', () => {
+    cy.visit('http://localhost:45000/iframe.html?id=core-ozon-content--extioref');
+
+    cy.get("dso-ozon-content").then((c) => {
+      c.prop("content", "<c><ExtIoRef xmlns='' href='https://identifier-eto.overheid.nl//join/id/regdata/pv25/2021/OKBebouwdEenOpHonderdWRIJ/nld@2021-11-14;1'>/join/id/regdata/pv25/2021/OKBebouwdEenOpHonderdWRIJ/nld@2021-11-14;1</ExtIoRef></c>");
+    });
+
+    cy.get("dso-ozon-content")
+      .find("span.fallback.od-c")
+      .children('a[href = "https://identifier-eto.overheid.nl//join/id/regdata/pv25/2021/OKBebouwdEenOpHonderdWRIJ/nld@2021-11-14;1"]')
+      .should("exist")
+      .children("span.sr-only")
+      .contains("opent in nieuw venster");
```

Comment:
* `cy.contains()` is een (scoped) selector zoals `cy.find()`, geen assertion.

Wat denk je van:

```suggestion
    cy.get("dso-ozon-content")
      .contains("/join/id/regdata/pv25/2021/OKBebouwdEenOpHonderdWRIJ/nld@2021-11-14;1")
      .should('have.prop', 'tagName', 'A')
      .and('have.attr', 'href', 'https://identifier-eto.overheid.nl//join/id/regdata/pv25/2021/OKBebouwdEenOpHonderdWRIJ/nld@2021-11-14;1')
      .contains('opent in nieuw venster')
      .should('exist');
```

Reply:
Dat ziet er goed uit.